gr-digital/python/digital/qa_packet_headerparser_b.py

```diff
@@ -16,6 +16,16 @@
 import pmt
 
 
+def _wait_for_msgs(precondition):
```

Comment:
Would it make sense to add this function as a method to `gr_unittest`?

Reply:
OK done.